pages/create/[chainId]/[collectionAddress].tsx

```
@@ -81,7 +81,7 @@ export const getServerSideProps = wrapServerSideProps<Props>(
       },
     })
     if (error) throw error
-    if (!data) throw new Error('data is falsy')
+    if (!data) return { notFound: true }
```

This one should throw an error as this `data` should always be present if we don't have an error. This is a typescript check only.
We could add a not found on the collection tho with `if (!data.collection) return { notFound: true }` additionally to the existing check

---

We have `Empty` state for this one, so we shouldn't throw `404` for this one.
Just changed it to throw `new Error` if there is no data 👍